packages/roosterjs-editor-api/lib/format/setListIndent.ts

```diff
@@ -0,0 +1,22 @@
+import { ChangeSource, IEditor } from 'roosterjs-editor-types';
+
+/**
+ * Set the Indent of a list
+ * @param editor The editor instance
+ * @param list The HTML element
+ * @param textIdent The indent value to be set to the list
+ * @param unit The Unit to be used.
+ */
+export default function setListIndent(
+    editor: IEditor,
+    list: HTMLElement,
+    textIdent: number,
+    unit: 'em' | 'px' | '%' | 'in'
+) {
+    editor.addUndoSnapshot((start, end) => {
+        editor.focus();
+        list.style.textIndent = textIdent.toString() + unit;
```

Comment:
Actually these two function are very similar. Then works in different way when this is a list.

And we already have another format API setIndentation. We need to think about whether we can merge these 3 together, and make it smart. For example, if a block already have a BLOCKQUOTE-based indentation, then user want to adjust its indentation by px, how should we handle it?

Reply:
We could use a object parameter on the setIndentation like this one, or add it as a not required parameter so we have backward compatibility

![image](https://user-images.githubusercontent.com/8291124/130809045-8277aee6-dee4-4bf6-9098-794a34d9d0f4.png)

And when any property of the object is not null set the styles on the element.